<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QPushButton" name="btnConnect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>210</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnSelectAll">
      <property name="text">
       <string>Select All</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lbGuest_id">
         <property name="text">
          <string>Гость</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="3">
        <widget class="QLineEdit" name="leguest"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbfull_name">
         <property name="text">
          <string>ФИО</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="6">
        <widget class="QLineEdit" name="lefull_name"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lbadress">
         <property name="text">
          <string>Адрес</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="5">
        <widget class="QLineEdit" name="leaddress"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="lbemail">
         <property name="text">
          <string>Почта</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="5">
        <widget class="QLineEdit" name="leemail"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="lbphone">
         <property name="text">
          <string>Телефон</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="4">
        <widget class="QLineEdit" name="lephone"/>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QLabel" name="lbcredit_card_number">
         <property name="text">
          <string>Кридитная карта</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5" colspan="2">
        <widget class="QLineEdit" name="lecredit_card_number"/>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4" colspan="2">
        <widget class="QPushButton" name="btnEdit">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="6" column="6">
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="btnDelete">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QTableWidget" name="twOrg"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTextEdit" name="teResult">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
